CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+#How to Contribute
+Below are the instructions for making changes to simple chat. There are a few important guidlines we'd wish for you to follow.
+These will make it easier for you to contribute and for us to follow. Thank you for contributing!
+
+##Getting Started
+If you found a bug there are two options. If you have a solution, please follow these steps:
+-Make sure you have a [Github account](https://github.com/join)
+-Fork the repository
+-Edit
+-Create a pull request
+Language used: TypeScript/JavaScript
+
+If you've found a bug and don't have a solution, please create an issue on our page to notify us. 
```

Comment:
We should mention feature requests here too. ☝️ 
You can leave the " on our page to notify us" out and just link the issue page like that:
`[create an issue](https://github.com/mbrandau/simple-chat/issues/new)`

Furthermore, I think it'd be better to have this line after line 6.